### PR TITLE
Unify SDK Version constant for all occurrences.

### PR DESCRIPTION
**Pull request by Rinnegatamante**

### CODE REVIEW

**Comment by d3m3vilurr** [comment]
@Rinnegatamante could you fix sample first? might be last vita2d patch would break the sdl2 sample.

**Comment by Rinnegatamante** [comment]
If the problem is about some undefined reference to SceSharedFb functions, it's case -lSceAppMgr_stub needs to be added to the libraries list.

**Comment by joel16** [comment]
Should be fixed with https://github.com/xerpi/libvita2d/pull/81#event-3489388796.

File: include/psp2/types.h [hunk 1]
```diff
@@ -7,6 +7,8 @@
 #ifndef _PSP2_TYPES_H_
 #define _PSP2_TYPES_H_
 
+#define PSP2_SDK_VERSION 0x03570011
```
Comment (d3m3vilurr) [inline include/psp2/types.h:10]: how about just make the another header like `def.h`?

Comment (Rinnegatamante) [reply include/psp2/types.h:10]: With a single define?

Comment (d3m3vilurr) [reply include/psp2/types.h:10]: yep. imo, a version couldn't be types.

**Pull request merged**